shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -70,6 +70,8 @@ public class ScanCommand extends Command {
   private Option contextOpt;
   private Option executionHintsOpt;
   private Option scanServerOpt;
+  protected Option showFewOpt;
+  private Option formatterOpt;
```

Comment:
I think this `formatterOpt` can stay in the list above. Only `showFewOpt` needed to be moved out to its own line, to make it `protected`.

Ultimately, we shouldn't have comma-separated lists of variables like these at all, so eventually, it'd be better if they were all in their own separate lines. But, to keep this PR minimal, it'd be good to move it back.